mne/connectivity/spectral.py

```diff
@@ -699,60 +699,47 @@ def spectral_connectivity(data, method='coh', indices=None, sfreq=2 * np.pi,
             C = ---------------------
                 sqrt(E[Sxx] * E[Syy])
 
-        'imcoh' : Imaginary coherence [1]_ given by::
+        'imcoh' : Imaginary coherence :footcite:`NolteEtAl2004`
+        given by::
 
                       Im(E[Sxy])
             C = ----------------------
                 sqrt(E[Sxx] * E[Syy])
-
-        'plv' : Phase-Locking Value (PLV) [2]_ given by::
-
+        'plv' : Phase-Locking Value (PLV)
+        :footcite:`LachauxEtAl1999` given by::
             PLV = |E[Sxy/|Sxy|]|
 
-        'ciplv' : corrected imaginary PLV (icPLV) [3]_ given by::
+        'ciplv' : corrected imaginary PLV (icPLV)
+        :footcite:`BruñaEtAl2018` given by::
```

Comment:
You have added the reference to `references.bib`, but the "citation key" that you added is now `BrunaEtAl2018`, without that special `ñ`

```suggestion
        :footcite:`BrunaEtAl2018` given by::
```